src/vip-framework/index.ts

```diff
@@ -16,7 +16,7 @@ let DELAY_BLOCKS = {
 };
 const VOTING_PERIOD = 28800;
 
-let DEFAULT_PROPOSER_ADDRESS = "0x55A9f5374Af30E3045FB491f1da3C2E8a74d168D";
+let DEFAULT_PROPOSER_ADDRESS = "0x97a32d4506f6a35de68e0680859cdf41d077a9a9";
```

Comment:
We don't need to change the default proposer. We can set specific proposers per VIP, as we did [here](https://github.com/VenusProtocol/vips/pull/85/files#diff-52c0128c8a7939d0fce846c4aef6b64fc4b59f80f6db5de8c2aba15dd8ec2111R272)